tests/client/core/i18n/test_utils.js

```diff
@@ -446,6 +446,7 @@ describe('i18n utils', () => {
     });
 
     it('localised formatting a number', () => {
+      // Skipped because node gives a different result.
```

Comment:
Nice catch -  I left the comment in by mistake after removing changing back from `it.skip()` -> `it()`